modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -121,7 +116,24 @@ data:
 
 ----
 +
-See the link:https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config[Prometheus relabel_config documentation] for information about write relabel configuration options.
+.Example of forwarding metrics called `my_metric_1` and `my_metric_2` in `my_namespace` namespace
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote-write-endpoint.example.com"
+        writeRelabelConfigs:
+        - sourceLabels: [__name__,namespace]
+          regex: '(my_metric_1|my_metric_2);my_namespace'
+          action: keep 
+----
 
 ** *To configure remote write for the Prometheus instance that monitors user-defined projects*:
```

Comment:
Is a fix for this and other bold text tracked anywhere?

Reply:
I am doing a big restructure of the monitoring docs that will solve that issue, because the two procedure will be separated. For more information, you can take a look at these two epics :slightly_smiling_face: 
https://issues.redhat.com/browse/OBSDOCS-1379
https://issues.redhat.com/browse/OBSDOCS-1267